bin/src/named.rs

```diff
@@ -271,124 +268,61 @@ async fn load_zone(
     Ok(authority)
 }
 
-// argument name constants for the CLI options
-const QUIET_ARG: &str = "quiet";
-const DEBUG_ARG: &str = "debug";
-const CONFIG_ARG: &str = "config";
-const ZONEDIR_ARG: &str = "zonedir";
-const PORT_ARG: &str = "port";
-const TLS_PORT_ARG: &str = "tls-port";
-const HTTPS_PORT_ARG: &str = "https-port";
-const QUIC_PORT_ARG: &str = "quic-port";
```

Comment:
About these one I was feeling that they weren't needed but maybe it's better to pass them to derive API.